prac_05/word_counting.py

```diff
@@ -1,3 +1,15 @@
 
-text = input("Text: ")
-words = text.split()
+
+def main():
+    word_bank = {}
+    user_text = input("Text: ").lower()
+    words = user_text.split()
+    for word in words:
+        if word in word_bank:
+            word_bank[word] += 1
+        else:
+            word_bank[word] = 1
+    for word, count in word_bank.items():
+        print("{:2} : {}".format(word, count))
+
+main()
```

Comment:
Now that the code works, try aligning the words neatly in the column (finding the largest word, like in the assignment). Also, blank line at the end of the file for PEP8 convention.